php/php-pcov/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
```

Comment:
you don't seem to be using the `github` PG; if that's the case please don't add it

Reply:
Ah, I don't know what the group implies, heh. Just saw it in some other ports and assumed it should be there if the repository sits @ GH. Sorry for that. I'be removed it in fixup commit.